lib/sync-records.ts

```diff
@@ -51,129 +49,127 @@ export const syncWorkbookRecords = async ({
     };
   }
 
-  const validJobs = await validJobRecords(jobRecords);
-
-  console.log("Valid job records to sync", validJobs.length);
-
-  const upsertJobs = await upsertJobRecords(validJobs, {
-    userId,
-    organizationId,
-  });
-
-  const validEmployees = await validEmployeeRecords(employeeRecords);
+  console.log("Valid job records to sync", numJobRecords);
 
-  console.log("Valid records to sync", validEmployees.length);
+  if (jobRecords) {
+    const upsertJobs = await upsertJobRecords(jobRecords, {
+      userId,
+      organizationId,
+    });
+  }
 
   // TODO: Refactor employee logic to its lib file
-  const validsManagersFirst = validEmployees.sort((a, b) => {
+  const validsManagersFirst = employeeRecords?.sort((a, b) => {
     return a.values.managerId.value ? 1 : -1;
   });
 
-  const upsertEmployees = validsManagersFirst.map(async (r) => {
-    try {
-      const values = r.values;
-      const employeeTypeId = (
-        (await prismaClient.employeeType.findUnique({
-          where: { slug: values.employeeType.value as string },
-        })) as EmployeeType
-      ).id;
-
-      let recordJobCode = () => {
-        if (values.jobCode && values.jobCode.value) {
-          return values.jobCode.value as string;
-        } else {
-          let jobName = values.jobName.value as string;
-
-          return jobName.replaceAll(" ", "_");
-        }
-      };
+  if (validsManagersFirst) {
```

Comment:
Why's this in an if? Shouldn't necessarily be required, and where's the else case?

Reply:
setting a console log here in case there are no employees to upsert